message LoginRequest{
}

message LoginResult{
	required int32 player_id = 2;
}
message LoginBroadcastResult{
	required int32 player_id = 2;
}